packages/components/style.css

```diff
@@ -1,5 +1,8 @@
-@import 'tailwindcss/base';
-@import 'tailwindcss/components';
+@layer l-base, l-nextra, l-components;
+
+@import 'tailwindcss/base' layer(l-base);
+@import 'nextra-theme-docs/dist/style.css' layer(l-nextra);
+@import 'tailwindcss/components' layer(l-components);
 @import 'tailwindcss/utilities';
```

Comment:
I've left `tailwindcss/uitilities` in the implicit layer, so the utility classes in React code can override any code in `.css` files with normal specificity like they do currently.